frontend/src/app/list/list.component.ts

```diff
@@ -91,20 +104,17 @@ export class ListComponent {
     this.onUploadFile = !this.onUploadFile;
     this.onUploadFile ? document.body.style.overflow = "hidden" : document.body.style.overflow = "auto";
     if (uploadSucess) {
-      /* NEED TO CHANGE after service done
-       * need to clear ${this.tags} and resfresh homePage if upload is suscess
-       */
+      // clear ${this.tags} and resfresh homePage if upload is suscess
```

Comment:
issue: typo--should be "refresh"

Reply:
fixed in the latest commit https://github.com/GoogleCloudPlatform/app-large-data-sharing-golang/pull/1/commits/6b656573d3db05f371011f3a0ae3f416e9b4fc74